src/internet_identity/internet_identity.did

```diff
@@ -519,6 +515,6 @@ service : (opt InternetIdentityInit) -> {
 
     // Attribute Sharing MVP API
     // The methods below are used to generate ID-alias credentials during attribute sharing flow.
-    prepare_id_alias : (PrepareIdAliasRequest) -> (opt PrepareIdAliasResponse);
-    get_id_alias : (GetIdAliasRequest) -> (opt GetIdAliasResponse) query;
+    prepare_id_alias : (PrepareIdAliasRequest) -> (variant {Ok: PreparedIdAlias; Err: PrepareIdAliasError;});
```

Comment:
Note: This PR removes the `opt` entirely. If we want to keep extensibility in the error only, is to just make the `Err` opt, or to add the `GenericError` as in ICRC-1.
@przydatek: Any preferences here?

Reply:
As discussed, let's leave as-is.